bloom-housing/bloom
# fix: remove dwy copy

This PR addresses #4338 


## Description

This change removes dwy specific lottery copy in the partner's lottery export terms (as approved by product). I left the Terms of Use string translations since Sarah said we would be using it soon for HBA's implementation.

## How Can This Be Tested/Reviewed?

Add a partner user to a lottery listing and attempt to export the lottery. You should see a reduced modal.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
